stubs/networkx/METADATA.toml

```
@@ -1,6 +1,7 @@
 version = "3.2.1"
 upstream_repository = "https://github.com/networkx/networkx"
-requires = ["numpy"]
+# TODO: Lots of stubtest errors when using numpy 2
+requires = ["numpy<2"]
```

We should probably require a version here that has a `py.typed` file, like we do for our other stubs, though that issue pre-exists this PR:

```suggestion
requires = ["numpy>=1.20,<2"]
```